packages/gatsby-plugin-mdx/README.md

```diff
@@ -110,6 +123,20 @@ layout defined, even if it's imported manually using `import MDX from './thing.m
 // gatsby-config.js
 module.exports = {
   plugins: [
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `pages`,
```

Comment:
Pages should be automatically sourced with MDX if there are any `.mdx` files. Were you not seeing this occur?

Reply:
Page were sourced, yes. But graphql node were not created for the pages, that's why I found it quite confusing, and that's why I think it's a good idea to include filesystem plugin in examples